src/main/java/org/ibp/api/brapi/v1/authorize/authorizeResourceBrapi.java

```diff
@@ -0,0 +1,29 @@
+package org.ibp.api.brapi.v1.authorize;
+
+import io.swagger.annotations.Api;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URI;
+import java.net.URLEncoder;
+
+@Api(value = "BrAPI authorize external Client Services")
+@Controller
+public class authorizeResourceBrapi {
+
+	@RequestMapping(value = "/brapi/authorize", method = RequestMethod.GET)
```

Comment:
Leaving a link here of where this is coming from:
https://github.com/PhenoApps/Field-Book/blob/c31637122e0a152d501b02d019e0298179602529/app/src/main/java/com/fieldbook/tracker/brapi/BrAPIService.java#L791
There's no spec for this yet.

Reply:
Probably we should move this method inside `AuthenticationControllerBrapi`, as the calls are closely related